playbook/app/pb_kits/playbook/pb_advanced_table/Components/CollapsibleTrail.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+
+//CollapsibleTrail component
+const CollapsibleTrail = ({ leftOffset }: { leftOffset: number }) => {
+  const style: { [key: string]: string | number } = {
+    position: "absolute",
+    left: `${leftOffset}rem`,
+    top: 0,
+    bottom: 0,
+    width: "2px",
+    backgroundColor: "#E4E8F0",
```

Comment:
Any chance to make these more sass driven? What about spacing props?

Reply:
I did try a version of this that used scss instead of inline styles on [this commit](https://github.com/powerhome/nitro-web/pull/36388/commits/357953529c92ec2dfabfa42a60241b3f6efbd06b) when I was still testing in the KPI table files, but there were several issues with that implementation. TLDR: since the collapsible trail is absolutely positioned, this way of handling the logic makes it more flexible and reduces unused scss shipping with the kit.